bot/main.py

```diff
@@ -0,0 +1,24 @@
+import discord
+
+intents = discord.Intents.default()
+intents.message_content = True
+
+client = discord.Client(intents=intents)
```

Comment:
please do not use client it is deprecated and doesn't have same features as bot

Reply:
To be honest, I have just copied a part of code from the documentation to have _**a running bot**_. I haven't looked into how the real bot should be implemented, and I am not planning to work on it - @HeraldPunisher will.